{
  "args": [
    "function 'Foo' redefined at line 4"
  ], 
  "exception_type": "<type 'exceptions.RuntimeError'>", 
  "stack": [
    [
      "<module>", 
      "got = ds.extractFromFile(os.path.join(testDir, test + \".js\"))"
    ], 
    [
      "extractFromFile", 
      "data = self.extract(contents)"
    ], 
    [
      "extract", 
      "self._analyzeBlock(block, context, firstBlock, stack, lineStart, line)"
    ], 
    [
      "_analyzeBlock", 
      "parseData['blockHandler'].merge(doc, stack[-1][1], guessedName, self._whatContext(stack))"
    ], 
    [
      "merge", 
      "raise RuntimeError(\"function '%s' redefined\" % doc[\"name\"])"
    ]
  ]
}
